crates/ruma-common/src/events/room/message.rs

```diff
@@ -458,6 +458,38 @@ impl MessageType {
         })
     }
 
+    /// A constructor to create a plain text message.
+    pub fn text_plain(body: impl Into<String>) -> Self {
```

Comment:
Nit: i feel like these should be swapped around, as "plain" counts as an adjective more, and "text" as a noun, respectively.

So it'd be "Plain Text" instead of "Text Plain".

Ditto with "HTML Text" and "Text HTML", I feel the former feels like more natural english

Reply:
"plain text" sounds like one thing though, when it's try to say "a text msgtype with plain(text) content".

Reply:
I also like that this way the different options are listed together in the docs' sidebar